charts/metallb/README.md

```diff
@@ -12,108 +12,108 @@ A network load-balancer implementation for Kubernetes using standard routing pro
-| prometheus.prometheusRule.annotations | object | `{}` |  |
-| prometheus.prometheusRule.bgpSessionDown.enabled | bool | `true` |  |
-| prometheus.prometheusRule.bgpSessionDown.labels.severity | string | `"alert"` |  |
-| prometheus.prometheusRule.configNotLoaded.enabled | bool | `true` |  |
-| prometheus.prometheusRule.configNotLoaded.labels.severity | string | `"warning"` |  |
-| prometheus.prometheusRule.enabled | bool | `false` |  |
-| prometheus.prometheusRule.extraAlerts | list | `[]` |  |
-| prometheus.prometheusRule.staleConfig.enabled | bool | `true` |  |
-| prometheus.prometheusRule.staleConfig.labels.severity | string | `"warning"` |  |
-| prometheus.scrapeAnnotations | bool | `false` |  |
-| prometheus.serviceAccount | string | `""` |  |
-| psp.create | bool | `true` |  |
-| rbac.create | bool | `true` |  |
-| speaker.affinity | object | `{}` |  |
-| speaker.enabled | bool | `true` |  |
-| speaker.frr.enabled | bool | `false` |  |
-| speaker.frr.image.pullPolicy | string | `nil` |  |
-| speaker.frr.image.repository | string | `"frrouting/frr"` |  |
-| speaker.frr.image.tag | string | `"v7.5.1"` |  |
-| speaker.image.pullPolicy | string | `nil` |  |
-| speaker.image.repository | string | `"quay.io/metallb/speaker"` |  |
-| speaker.image.tag | string | `nil` |  |
-| speaker.livenessProbe.enabled | bool | `true` |  |
-| speaker.livenessProbe.failureThreshold | int | `3` |  |
-| speaker.livenessProbe.initialDelaySeconds | int | `10` |  |
-| speaker.livenessProbe.periodSeconds | int | `10` |  |
-| speaker.livenessProbe.successThreshold | int | `1` |  |
-| speaker.livenessProbe.timeoutSeconds | int | `1` |  |
-| speaker.logLevel | string | `"info"` | Speaker log level. Must be one of: `all`, `debug`, `info`, `warn`, `error` or `none` |
-| speaker.memberlist.enabled | bool | `true` |  |
-| speaker.memberlist.mlBindPort | int | `7946` |  |
-| speaker.nodeSelector | object | `{}` |  |
-| speaker.podAnnotations | object | `{}` |  |
-| speaker.priorityClassName | string | `""` |  |
-| speaker.readinessProbe.enabled | bool | `true` |  |
-| speaker.readinessProbe.failureThreshold | int | `3` |  |
-| speaker.readinessProbe.initialDelaySeconds | int | `10` |  |
-| speaker.readinessProbe.periodSeconds | int | `10` |  |
-| speaker.readinessProbe.successThreshold | int | `1` |  |
-| speaker.readinessProbe.timeoutSeconds | int | `1` |  |
-| speaker.resources | object | `{}` |  |
-| speaker.runtimeClassName | string | `""` |  |
-| speaker.serviceAccount.annotations | object | `{}` |  |
-| speaker.serviceAccount.create | bool | `true` |  |
-| speaker.serviceAccount.name | string | `""` |  |
-| speaker.tolerateMaster | bool | `true` |  |
-| speaker.tolerations | list | `[]` |  |
+| Key                                                                      | Type   | Default                        | Description                                                                             |
```

Comment:
This is autogenerated and we have CI enforcing, so I'd avoid changing or CI will never pass